Sources/Testing/Running/XCTestScaffold.swift

```diff
@@ -158,12 +158,11 @@ public enum XCTestScaffold: Sendable {
         testCase.rawValue.record(XCTIssue(issue, processLaunchedByXcode: isProcessLaunchedByXcode))
       }
 #else
-      // NOTE: XCTestCase.recordFailure(withDescription:inFile:atLine:expected:)
```

Comment:
Please keep this comment here so that we don't accidentally misuse the expected parameter later.